main.go

```diff
@@ -225,8 +225,18 @@ func runCLIApp(c *cli.Context) (err error) {
 	}
 
 	logger.Infof("Start gcsfuse/%s for app %q using mount point: %s\n", getVersion(), flags.AppName, mountPoint)
-	logger.Infof("GCSFuse mount command flags: %s", util.Stringify(*flags))
-	logger.Infof("GCSFuse mount config flags: %s", util.Stringify(*mountConfig))
+
+	flagsStringified, err := util.Stringify(*flags)
+	if err != nil {
+		logger.Warnf("failed to stringify cli flags: %v", err)
```

Comment:
We should use %w to format error. 
Ref: https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w

Reply:
gcsfuse/main.go:231:3: github.com/googlecloudplatform/gcsfuse/internal/logger.Warnf does not support error-wrapping directive %w

I tried this earlier too. Looks like %w is supported only for fmt.Error as that's returning an error by wrapping another, not for convering it to a string.